config/config_test.go

```diff
@@ -169,3 +169,28 @@ xkNuJ2BlEGkwWLiRbKy1lNBBFUXKuhh3L/EIY10WTnr3TQzeL6H1
 	c.Assert(os.Remove(certFile), IsNil)
 	c.Assert(os.Remove(keyFile), IsNil)
 }
+
+func (s *testConfigSuite) TestConfigDiff(c *C) {
+	c1 := NewConfig()
+	c2 := &Config{}
+	*c2 = *c1
+	c1.Host = "c1"
+	c2.Host = "c2"
+	c1.Log.Format = "c1"
```

Comment:
We do not support hot-reload Host and Log.Format now, maybe we can change these cases as `OOMAction` and `MemQuotaQuery` to avoid confusing the readers.

Reply:
Yes you are right and it has been updated now.